client.go

```diff
@@ -159,6 +158,10 @@ func (c *Client) sendScalingMsg(data *logMetrics) {
 		"prefix": *data.prefix,
 	}).Debug("sendScalingMsg")
 
+	for _, v := range data.events {
+		c.SimpleEvent("Heroku app: "+*data.app, v)
```

Comment:
In that case maybe better to put `heroku/api` as maybe in the future there will be some event from `heroku/router` or something like that so you know from which part of heroku it comes? what do you think?